velox/common/memory/MemoryPool.cpp

```diff
@@ -534,7 +543,6 @@ const std::vector<MachinePageCount>& MemoryPoolImpl::sizeClasses() const {
 void MemoryPoolImpl::allocateContiguous(
     MachinePageCount numPages,
     ContiguousAllocation& out) {
-  CHECK_AND_INC_MEM_OP_STATS(Allocs);
```

Comment:
I felt like we might just keep stats update at the function entrance otherwise, the code looks little messy and error prone. We probably shall add another counter call allocationError? and we can just assume all kinds of allocation free always succeed?